oauth2-server/src/test/java/com/clouway/oauth2/IssueNewTokenForClientTest.java

```diff
@@ -94,20 +94,13 @@ public void tokenCannotBeIssued() throws Exception {
 
       oneOf(identityFinder).findIdentity(with(any(String.class)), with(any(GrantType.class)), with(any(DateTime.class)));
       will(returnValue(Optional.of(identity)));
-
```

Comment:
Keep in mind that signing need to be performed by private keys and verification to be performed with public keys, which means that name of the method is misleading and requires some fixes.